.amethyst.sample.yml

```diff
@@ -1,42 +1,280 @@
-# Use tall and fullscreen layouts
+# Default settings for amethyst
+# Repo: `https://github.com/ianyh/Amethyst`
+#
+# Move this file to: `~/.amethyst.yml`
+# In order to register changes restart Amethyst.
+# If you experience issues pulling in the changes you can also quit Amethyst and run: `rm ~/Library/Preferences/com.amethyst.Amethyst.plist`
+# This removes the current preferences and causes a re-load of this file when amethyst restarts.
```

Comment:
I don't know all the mechanics behind macOS and preferences, but I have been playing with preferences yesterday for thie issue #1348  and the `/Library/Preferences/com.amethyst.Amethyst.plist` file does not seem to always be generated (at least on my Mac mini macOS Ventura). However, you can reset your preferences using `defaults delete com.amethyst.Amethyst.plist`

```suggestion
# If you experience issues pulling in the changes you can also quit Amethyst and run: `defaults delete com.amethyst.Amethyst.plist`
# This removes the current preferences and causes Amethyst to restart with default preferences and pull configs from this file.
```